src/test/resources/application.yml

```diff
@@ -0,0 +1,145 @@
+server:
+  port: 9100
+  contextPath: 
+
+# use this as the security key for encryption
+security:
+  user:
+    password: 81d56928-891a-41b2-8dd2-132593e6d057
+
+spring:
+  main:
+    allow-bean-definition-overriding: true
+    allow-circular-references: true
+  template:
+    cache: false
+  http:
+    multipart:
+      maxFileSize: 100MB
+      maxRequestSize: 100MB
+  sql:
+    init:
+      platform: h2
+  datasource:
+    url: jdbc:h2:mem:AZ;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    driverClassName: org.h2.Driver
+    jpa:
+      database-platform: org.hibernate.dialect.H2Dialect
+    username: cap
+    password: cap
+    # common database config -- might be needed for postgres
+    defaultAutoCommit: false
+    test-on-borrow: true
+    validation-query: select version();
+  jpa:
+    generate-ddl: true
+    show-sql: false
+    open-in-view: true
+    hibernate:
+      ddl-auto: create-drop
+      event.merge.entity_copy_observer: allow
+  # TODO: figure out how to use profiles in Spring-Boot 2.6.6
+  # profiles: test
+  # messaging
+  # to enable messaging, include profile weaver-messaging and update activemq broker-url, username and password
+  activemq:
+    broker-url: tcp://localhost:61616
+    user: username
+    password: password
+
+multipart:
+  # File size limit
+  maxFileSize: 100Mb
+  # Total request size for a multipart/form-data
+  maxRequestSize: 1000Mb
+
+# logging
+logging:
+  level:
+    edu:
+      tamu: INFO
+    org:
+      hibernate: INFO
+      springframework: INFO
+    ro:
+      isdc:
+        wro: INFO
+  file.name: logs/cap.log
```

Comment:
I started a practice of adding `-test` to the test log file.
I request changing this to:
```
file.name: logs/cap-test.log
```